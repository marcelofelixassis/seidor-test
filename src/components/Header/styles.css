.header {
    background-color: #07070d;
    width: calc(100% - 20px);
    height: 80px;
    padding: 10px;
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: space-between;

}

.title {
    color: white;
    font-size: 16px;
}