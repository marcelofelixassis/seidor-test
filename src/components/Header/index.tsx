import AttachMoneyIcon from '@material-ui/icons/AttachMoney';
import React from 'react';
import './styles.css';

const Header: React.FC = () => (
    <div className="header">
        <AttachMoneyIcon style={{color: "white"}} />
        <span className="title">Registrar Funcionario - Tabelas e c√°lculos do IRRF</span>
    </div>
)

export default Header;
