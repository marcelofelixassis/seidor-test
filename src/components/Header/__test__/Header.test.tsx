import React from 'react';
import { shallow } from 'enzyme';
import Header from '../index';
test('renders the component', () => {
  const component = shallow(<Header />);
  expect(component).toMatchSnapshot();
});