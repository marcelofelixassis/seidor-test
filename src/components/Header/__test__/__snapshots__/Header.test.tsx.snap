// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<div
  className="header"
>
  <AttachMoneyIcon
    style={
      Object {
        "color": "white",
      }
    }
  />
  <span
    className="title"
  >
    Registrar Funcionario - Tabelas e c√°lculos do IRRF
  </span>
</div>
`;
